setup.py

```diff
@@ -1,3 +1,7 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
```

Comment:
Case in point, this is pretty useless here.

Reply:
[62e01c1](https://github.com/PyCQA/astroid/pull/1441/commits/62e01c19ae4a90503d1f3212c062a44ac347cd35)